examples/basic/variables.tf

```diff
@@ -41,13 +41,9 @@ variable "existing_cluster_name" {
   description = "Existing cluster name"
   type        = string
   default     = null
-  validation {
-    condition     = can(regex("^[a-z][a-z0-9-]{0,12}[a-z0-9]$", var.existing_cluster_name))
-    error_message = "Existing cluster name must begin with a letter and contain only lowercase letters, numbers, and - characters. Existing cluster names must end with a lowercase letter or number and be 13 or fewer characters."
-  }
 }
 
-variable "existing_cluster_rg_id" {
+variable "existing_resource_group_name" {
```

Comment:
Can you keep consistent with how we do all other examples. Takes in resource group name and supports creating a new one if value is null